### ISSUE TITLE
Laravel 5.6 Support

### ISSUE THREAD

**Issue by chaseconey** [swisnl/json-api-client#24]
As of right now it looks like this works up to Laravel 5.5. When trying to use 5.6, getting an error about the version of `support` being used (see below). Any plans on upping the version requirement on that?

```
Using version ^0.8.0 for swisnl/json-api-client
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Installation request for swisnl/json-api-client ^0.8.0 -> satisfiable by swisnl/json-api-client[0.8.0].
    - Conclusion: remove laravel/framework v5.6.34
    - Conclusion: don't install laravel/framework v5.6.34
    - swisnl/json-api-client 0.8.0 requires illuminate/support 5.3.*|5.4.*|5.5.* -> satisfiable by laravel/framework[5.4.x-dev, 5.5.x-dev], illuminate/support[5.3.x-dev, 5.4.x-dev, 5.5.x-dev, v5.3.0, v5.3.16, v5.3.23, v5.3.4, v5.4.0, v5.4.13, v5.4.17, v5.4.19, v5.4.27, v5.4.36, v5.4.9, v5.5.0, v5.5.16, v5.5.17, v5.5.2, v5.5.28, v5.5.33, v5.5.34, v5.5.35, v5.5.36, v5.5.37, v5.5.39, v5.5.40, v5.5.41].
    - Can only install one of: laravel/framework[5.4.x-dev, v5.6.34].
    - Can only install one of: laravel/framework[5.5.x-dev, v5.6.34].
    - illuminate/support 5.3.x-dev conflicts with laravel/framework[v5.6.34].
    - illuminate/support 5.4.x-dev conflicts with laravel/framework[v5.6.34].
    - illuminate/support 5.5.x-dev conflicts with laravel/framework[v5.6.34].
    - illuminate/support v5.3.0 conflicts with laravel/framework[v5.6.34].
    - illuminate/support v5.3.16 conflicts with laravel/framework[v5.6.34].
    - illuminate/support v5.3.23 conflicts with laravel/framework[v5.6.34].
    - illuminate/support v5.3.4 conflicts with laravel/framework[v5.6.34].
    - illuminate/support v5.4.0 conflicts with laravel/framework[v5.6.34].
    - illuminate/support v5.4.13 conflicts with laravel/framework[v5.6.34].
    - illuminate/support v5.4.17 conflicts with laravel/framework[v5.6.34].
    - illuminate/support v5.4.19 conflicts with laravel/framework[v5.6.34].
    - illuminate/support v5.4.27 conflicts with laravel/framework[v5.6.34].
    - illuminate/support v5.4.36 conflicts with laravel/framework[v5.6.34].
    - illuminate/support v5.4.9 conflicts with laravel/framework[v5.6.34].
    - illuminate/support v5.5.0 conflicts with laravel/framework[v5.6.34].
    - illuminate/support v5.5.16 conflicts with laravel/framework[v5.6.34].
    - illuminate/support v5.5.17 conflicts with laravel/framework[v5.6.34].
    - illuminate/support v5.5.2 conflicts with laravel/framework[v5.6.34].
    - illuminate/support v5.5.28 conflicts with laravel/framework[v5.6.34].
    - illuminate/support v5.5.35 conflicts with laravel/framework[v5.6.34].
    - illuminate/support v5.5.36 conflicts with laravel/framework[v5.6.34].
    - illuminate/support v5.5.37 conflicts with laravel/framework[v5.6.34].
    - illuminate/support v5.5.39 conflicts with laravel/framework[v5.6.34].
    - illuminate/support v5.5.40 conflicts with laravel/framework[v5.6.34].
    - illuminate/support v5.5.41 conflicts with laravel/framework[v5.6.34].
    - don't install illuminate/support v5.5.33|don't install laravel/framework v5.6.34
    - don't install illuminate/support v5.5.34|don't install laravel/framework v5.6.34
    - Installation request for laravel/framework (locked at v5.6.34, required as 5.6.*) -> satisfiable by laravel/framework[v5.6.34].
```

**Comment by JaZo** [comment]
Hi @chaseconey we do plan to support Laravel 5.6 in the near future. However, you are free to submit a PR with Laravel 5.6 support to speed things up.

**Comment by JaZo** [comment]
Hi @chaseconey we have just released [version 0.9.0](https://github.com/swisnl/json-api-client/releases/tag/0.9.0) which includes support for Laravel 5.6!
